packages/react-scripts/template/README.md

```diff
@@ -936,6 +937,53 @@ Install the Surge CLI if you haven't already by running `npm install -g surge`.
 
 Note that in order to support routers that use html5 `pushState` API, you may want to rename the `index.html` in your build folder to `200.html` before deploying to Surge. This [ensures that every URL falls back to that file](https://surge.sh/help/adding-a-200-page-for-client-side-routing).
 
+## Forking as Alternative to Ejecting
+You can fork `react-scripts`, publish your fork, and then use it with Create React App.
+
+Pros:
+- You can customize your setup (eg: add CSS Modules, PostCSS plugins, decorators, etc…)
+- You still have all Create React App features and updates
+
+Cons:
+- You need to maintain your fork, [make sure it is synced](https://help.github.com/articles/fork-a-repo/#keep-your-fork-synced) with the upstream to have all updates. [Backstroke](https://github.com/1egoman/backstroke) is a bot that can help you with this.
```

Comment:
It might be worth linking to [this comment](https://github.com/facebookincubator/create-react-app/issues/682#issuecomment-268986340) explaining that the "classic" webhook method should be used if one wants Backstroke to assist in keeping the fork synced.